docs/rules.md

```diff
@@ -76,13 +78,24 @@ Checks the existence of a given file.
 
 | Input                  | Required | Type       | Default  | Description                                                                                                                                                                  |
-| ---------------------- | -------- | ---------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ---------------------- | -------- | ---------- | -------- |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `globsAny`             | **Yes**  | `string[]` |          | A list of globs to search for. This rule passes if at least one file found matches the provided hash, and fails if no files are found.                                       |
-| `hash`                 | **Yes**  | `string`   |          | The hash to check against. Unless a different `algorithm` is specified, this will be sha256.                                                                                 |
+| `hash`                 | **Yes**  | `string`   |          | The hash to check against. Unless a different `algorithm` is specified, this will be sha256 hash.                                                                            |
 | `algorithm`            | No       | `string`   | `sha256` | The hash algorithm to use. Repolinter supports any algorithm supported by [crypto.createHash](https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm_options) |
 | `nocase`               | No       | `boolean`  | `false`  | Set to `true` to perform an case insensitive search.                                                                                                                         |
 | `succeed-on-non-exist` | No       | `boolean`  | `false`  | Set to `true` to enable passing if no files are found from `globsAll`.                                                                                                       |
 
+### `file-hashes-not-exist`
+
+Check files' hashes not included in a list of certain cryptographic hashes.
+
+| Input                  | Required | Type       | Default  | Description                                                                                                                                                                  |
+|------------------------| -------- |------------|----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `globsAll`             | **Yes**  | `string[]` |          | A list of globs to search for. This rule passes if all of the markup files found do not contain broken links.                                                                |
```

Comment:
broken links?

Reply:
I just realize all `globsAll` sections in rules are different. I'll update them to be more meaningful.